<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>270</y>
     <width>593</width>
     <height>175</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="plot_input_output_layout">
    <item>
     <widget class="QTabWidget" name="plot_table">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="RX_signal_wave">
       <attribute name="title">
        <string>RX_Sig_wave</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="TX_signal_wave">
       <attribute name="title">
        <string>TX_Sig_wave</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>8</y>
     <width>89</width>
     <height>259</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="set_window_layout">
    <item>
     <widget class="QComboBox" name="set_ports_box">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="set_baud_rate_box">
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="set_parity_box">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="set_data_bit_box">
      <item>
       <property name="text">
        <string>8bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5bit</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="set_stop_bit_box">
      <item>
       <property name="text">
        <string>1bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2bit</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="set_ports_label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>34</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ports</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_baud_rate_label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>82</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BRate</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_parity_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_data_bit_label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>178</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DataBit</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_stop_bit_label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>228</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>StopBit</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>6</y>
     <width>415</width>
     <height>285</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="input_output_layout">
    <item row="3" column="0">
     <widget class="QPushButton" name="start_button">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="output_window"/>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="input_window"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="sent_data_button">
      <property name="text">
       <string>Sent Data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="clear_rx_button">
      <property name="text">
       <string>Clear RX</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="clear_tx_button">
      <property name="text">
       <string>Clear TX</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="stop_button">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
